### PR TITLE
Fixed  class sort api url [#177438813]

### PR DESCRIPTION
**Pull request by dougmartin**
The class sort api url was set as a member resource in the routes which was not correct (as it pertains to all classes and not a specific class).

This caused the "Manage Class List" to silently fail which wasn't evident until you reloaded the page and saw the class orderings revert.

### CODE REVIEW

File: rails/config/routes.rb [hunk 1]
```diff
@@ -443,11 +443,13 @@
...
         resources :teacher_classes, only: [:show] do
           member do
-            post :sort
             post :set_active
             post :copy
           end
         end
+        namespace :teacher_classes do
+          post :sort
+        end
```
Comment (scytacki) [inline rails/config/routes.rb:452]: Did you try add this as a collection inside of the teacher_classes resource?
Like 
`get :available_collaborators` above
Perhaps doing that will add some other stuff that will break the route, but if it works, it looks cleaner.

**Review by scytacki — CHANGES_REQUESTED** [review]
try adding as a collection

File: rails/config/routes.rb (hunk 1 above)
Comment (dougmartin) [reply rails/config/routes.rb:452]: Yes, I tried that first and it didn't work so I switched to suing a namespace like the classes route above it uses.

Comment (dougmartin) [reply rails/config/routes.rb:452]: Actually I tried it as a bare `post` - I'll try the collection approach.

**Review by scytacki — APPROVED** [review]
Looks good

**Pull request merged**